fundingmanager.go

```diff
@@ -1821,6 +1824,7 @@ func (f *fundingManager) handleFundingSigned(fmsg *fundingSignedMsg) {
 				OutputIndex: fundingPoint.Index,
 			},
 		},
+		PendingChanId: pendingChanID[:],
```

Comment:
Should we add the `PendingChanId` to `stateStep` (line 931 ff) also? So the caller knows it's their channel that is now fully open and they can clean up the ship on their side.

Reply:
Done.